plugins/CoreHome/vue/src/Widget/Widgets.store.ts

```diff
@@ -66,11 +66,9 @@ class WidgetsStore {
   }
 
   reloadAvailableWidgets(): Promise<WidgetsStore['widgets']['value']> {
-    if (typeof window.widgetsHelper === 'object' && window.widgetsHelper.availableWidgets) {
```

Comment:
This never evaluated to true because window.widgetsHelper is defined as a function here: https://github.com/matomo-org/matomo/blob/5.x-dev/plugins/Dashboard/javascripts/widgetMenu.js#L8-L7